src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -458,7 +459,7 @@ public function iHaveAlreadyPlacedOrderNthTimes(
     }
 
     /**
-     * @Given there is an :orderNumber order with :product product
+     * @Given there is a(n) :orderNumber order with :product product
```

Comment:
I'd go without that change as it adds a bit of complexity to the term here, but no real value in the feature file